src/Tribe/Attendees.php

```diff
@@ -921,8 +925,11 @@ public function send_mail_list( $event_id = null, ?string $type = 'user', $send_
 			return $error;
 		}
 
+		// Send to could be an email or a user ID.
+		$email = sanitize_email( $send_to );
+
 		if ( 'user' === $type ) {
-			$user = get_user_by( 'id', $send_to );
+			$user = get_user_by( 'id', (int) $send_to );
```

Comment:
What if `$send_to` is null? We should cover that possibility before we get here and throw an error

Reply:
If `$send_to` is `null`, the casting it to an `int` would resolve to `0` which would result in `false` being returned instead of ` `WP_User`. The `Invalid User ID` error would be thrown in this case, which is exactly what we would want.

Reply:
OK, just wanted to be sure we were considering the result.